src/Racing.js

```diff
@@ -0,0 +1,37 @@
+import { Console, Random } from '@woowacourse/mission-utils';
+import { MIN_RANDOM_SCORE } from './constants/index.js';
+
+class Racing {
+  #totalRound = 0;
+
+  #racers = [];
+
+  constructor({ totalRound, racers }) {
+    this.#totalRound = totalRound;
+    this.#racers = racers;
+  }
+
+  start() {
+    for (let i = 0; i < this.#totalRound; i += 1) {
+      this.#moveForward();
+      Console.print(this.#printResultByRound());
+    }
+    return this;
+  }
+
+  #moveForward() {
+    this.#racers.forEach((racer) => {
+      if (Random.pickNumberInRange(0, 9) >= MIN_RANDOM_SCORE) {
+        racer.addScore();
+      }
+    });
+  }
+
+  #printResultByRound() {
+    return this.#racers
+      .map((racer) => `${racer.getName()} : ${racer.getRaceLine()}`)
+      .join('\n');
+  }
```

Comment:
해당 메서드에 `print`라는 네이밍이 있는만큼 `Console.print`도 여기서 수행하면 어떨까요?
```suggestion
  #printResultByRound() {
    const result = this.#racers
      .map((racer) => `${racer.getName()} : ${racer.getRaceLine()}`)
      .join('\n');
    Console.print(result);
  }
```
이런 식으로요!

지금처럼 `print` 기능을 안하고 프린트할 문자열을 반환하는 메서드라면 `getResultByRound()`가 더 의도를 전달하기 용이할 것 같아요!